app/assets/stylesheets/pages/_show.scss

```diff
@@ -18,3 +18,24 @@ body {
 .google-review {
   display: inline;
 }
+
```

Comment:
This refers to clicking on the flags and adding the rating to the db. This works, but we don't update the flag images and we removed all of this functionality from the show page due to merge conflicts. Figure out how to include this.